scripts/stats/global_ens/exevs_global_ens_atmos_grid2grid_stats.sh

```diff
@@ -87,11 +87,11 @@ if [ $ens = all ] || [ $ens = gefs ] || [ $ens = cmce ] || [ $ens = naefs ] || [
 
      elif [ $ens = naefs ] && [ $verif_case = upper ] ; then
 
-      if [ ! -s ${COMIN}.${VDATE}/cmce/cmcanl.t00z.grid3.f000.grib2 ] || [ ! -s ${COMIN}.${VDATE}/gefs/gfsanl.t00z.grid3.f000.grib2 ] ; then
+      if [ ! -s ${EVSIN}.${VDATE}/cmce/cmcanl.t00z.grid3.f000.grib2 ] || [ ! -s ${EVSIN}.${VDATE}/gefs/gfsanl.t00z.grid3.f000.grib2 ] ; then
```

Comment:
@BinbinZhou-NOAA , can you correct this line to be a check for naefs data? Can you confirm that this line is correct, too?

Reply:
Hi @BinbinZhou-NOAA , what about line 90 in exevs_global_ens_atmos_grid2grid_stat.sh? Please check that one, too. I see a test on cmce data in the naefs if-block.